app/contexts/services/imdb/search.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Contexts
+  module Services
+    module Imdb
+      class Search
+        attr_reader :clients, :api_key, :url, :lang
+
+        def initialize(config: {client: HTTParty })
+          @clients = {
+            api: config[:client],
+            imdb_client: Contexts::Services::Imdb::Client.new
+          }
+        end
+
+        def standarize(url:)
+          url.gsub(' ', '%20')
+        end
+
+        def call(expression: '')
+          expression = standarize(url: expression)
+          clients[:api].get(clients[:imdb_client].url + expression)
```

Comment:
did you try to send request without substituting spaces? 
I suppose it should work
I hope that HTTParty will escape spaces under the hood, isn't it?  
```suggestion
          clients[:api].get(clients[:imdb_client].url + expression)
```